src/Schema/Keywords/UniqueItems.php

```diff
@@ -25,17 +24,23 @@ class UniqueItems extends BaseKeyword
      * value false.
      *
      * @param mixed $data
+     *
+     * @throws ValidationKeywordFailed
      */
     public function validate($data, bool $uniqueItems) : void
     {
+        if (! $uniqueItems) {
```

Comment:
Also small bugfix. `uniqueItems` could be set to `false` explicitly

Reply:
Excellent catch!